src/index.html

```diff
@@ -4,13 +4,34 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-container">
+      <form
+        class="search-bar big-form"
+        data-qa="big"
+        method="get"
+        action="/">
```

Comment:
Need to check and fix it everywhere
<img width="1022" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/4615563d-5369-48a6-a9fe-ad6cc7c10b8f">
```suggestion
      <form
        class="search-bar big-form"
        data-qa="big"
        method="get"
        action="/"
      >
```